plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1233,20 +1233,28 @@ blockly.block.block_predicate_mutator.input=Condition
 blockly.block.block_predicate_not=Not %1
 blockly.block.block_predicate_replaceable=Block at position offset by x: %1 y: %2 z: %3 is of replaceable material
 blockly.block.block_predicate_solid=Block at position offset by x: %1 y: %2 z: %3 is of solid material
+blockly.block.block_predicate_true=True
 blockly.block.block_predicate_would_survive=%1 would survive at position offset by x: %2 y: %3 z: %4
+blockly.block.feature_bamboo=Bamboo feature with %1 chance of podzol disk
 blockly.block.feature_block_pile=Block pile feature with block: %1
+blockly.block.feature_block_column_simple=Simple block column feature with height: %1 block: %2
 blockly.block.feature_coral_claw=Underwater coral claw feature
 blockly.block.feature_coral_mushroom=Underwater coral mushroom feature
 blockly.block.feature_coral_tree=Underwater coral tree feature
+blockly.block.feature_delta=Delta feature with contents: %1 rim: %2 size: %3 rim size: %4
 blockly.block.feature_forest_rock=Forest rock feature with block: %1
 blockly.block.feature_huge_brown_mushroom=Huge brown mushroom feature with cap: %1 stem: %2 radius: %3
 blockly.block.feature_huge_red_mushroom=Huge red mushroom feature with cap: %1 stem: %2 radius: %3
 blockly.block.feature_huge_fungus=Huge fungus feature with stem: %2 hat: %3 decor: %4 planted: %5, only on ground: %1
 blockly.block.feature_lake=Lake feature with fluid: %1 border: %2
+blockly.block.feature_random_patch_simple=Simple patch feature with block: %4 condition: %5 number of tries: %1 XZ spread: %2 Y spread: %3
+blockly.block.feature_replace_sphere=Replace blob feature with target blockstate: %1 new blockstate: %2 radius: %3
```

Comment:
I think better wording would be

`Blob feature replacing: %1 with : %2 radius: %3`